app/bundles/InstallBundle/Translations/en_US/messages.ini

```diff
@@ -60,6 +60,7 @@ mautic.install.function.sessionstart="Install and enable the <strong>session</st
 mautic.install.function.simplexml="Install and enable the <strong>SimpleXML</strong> extension."
 mautic.install.function.tokengetall="Install and enable the <strong>Tokenizer</strong> extension."
 mautic.install.function.xml="Install and enable the <strong>XML</strong> extension."
+mautic.install.memory.limit="The memory_limit setting in php.ini is lower than suggested minimal limit of 128MB. Mautic can have troubles to process larger datasets and even to install."
```

Comment:
Try this instead:

`The <strong>memory_limit</strong> setting in your PHP configuration is lower than the suggested minimum limit of 128MB. Mautic can have performance issues with large datasets without sufficient memory.`